### ISSUE TITLE
unable to open Options button in AVG 2014

### ISSUE THREAD

**Issue by nvaccessAuto** [nvaccess/nvda#4604]
**Reported by blindbhavya on 2014-11-07 06:18**
This issue occurs with NVDA and not with JAWS.
Steps to reproduce
Open AVG anti virus fre edition 2014.
Tab to the Options button and press Enter.
Expected: The Options dialog or something should open (as it opens with JAWS).
Actual: Nothing really opens.
Ideally, this issue should be with AVG, but surprisingly, it worked with JAWS, and didn't with NVDA, so I filed this ticket.

**Comment by ehollig** [comment]
From the issue reported in #5120, it sounds as if this issue has been resolved in AVG 2015 but information is not readable from within the options dialog. Closing
